src/components/destination/index.js

```diff
@@ -32,8 +34,8 @@ class DestinationFullPage extends Component {
             googleMapElement={
               <GoogleMap
                 ref={(map) => (map) => console.log(map)}
-                defaultZoom={5}
-                defaultCenter={{ lat: 40.41, lng: -3.7 }}
+                defaultZoom={6}
+                defaultCenter={{ lat: latitude, lng: longitude }}
```

Comment:
you could just use `marker.position` here 

i.e.  `defaultCenter={marker.position}`